tests/int_tests.py

```diff
@@ -16,6 +20,26 @@ def _remove_file(path):
         pass
 
 
+def run(
+    args: List[str], outdir: str, outfmt: OutFmt = OutFmt.AUTO
```

Comment:
args can include both positional and non-positional arguments. `args` can include all  the arguments to the function, but outdir and outfmt are helpers to make it easier to specify those since they're common.

Reply:
deferring this to a potential issue: #58